galaxy/Dockerfile

```diff
@@ -125,6 +125,7 @@ RUN ansible-playbook /ansible/postgresql_provision.yml && \
     --extra-vars galaxy_extras_config_condor=True \
     --extra-vars galaxy_extras_config_condor_docker=True \
     --extra-vars galaxy_extras_config_rabbitmq=True \
+    --extra-vars galaxy_extras_config_cvmfs=True \
```

Comment:
Do you have commit access to his branch? He said he's away from computer about 40 minutes ago.